tests/commands/json/map/test_cached_map_info.py

```diff
@@ -85,12 +102,12 @@ async def test_getCachedMapInfo(
                     ),
                 }
             ),
-            *[firmware_event for _ in MapSetType],
+            *[firmware_event for _ in range(len(expected_commands))],
```

Comment:
```suggestion
            *[firmware_event for _ in expected_commands],
```
this should do the same without overhead for calculating length and range thereof

thanks for the PR!